datadog-agent.yaml

```diff
@@ -54,9 +54,16 @@ pipeline:
       tag: ${{package.version}}
       expected-commit: 024f4fb4b528f0eabaeeb4114744dd63edbe3553
 
+  - runs: |
+      go mod edit -dropreplace=github.com/moby/buildkit
```

Comment:
I *think* gobump should be smart enough to drop the replace on its own

Reply:
for some reason, the gobump was not dropping the replace instead updating it
```patch
- replace github.com/moby/buildkit v0.11.0 => github.com/moby/buildkit v0.11.4
+ replace github.com/moby/buildkit => github.com/moby/buildkit v0.13.1
````
It may still work without dropping, I didn't try though, let me check